gst/isomp4/qtdemux.c

```diff
@@ -3004,7 +2921,6 @@ qtdemux_parse_moof (GstQTDemux * qtdemux, const guint8 * buffer, guint length,
     /* On encrypted streams emit a signal with the senc node */
     if (stream->encrypted) {
```

Comment:
In case of disabled fluc-sdk we're not supporting encrypted files. So we should write some kind of error log, and exit with error, but now we have a broken code for case "w/o flucsdk" + "encrypted file"